models/basic_rover/model.config

```diff
@@ -1,11 +0,0 @@
-<?xml version="1.0" ?>
```

Comment:
I think this file should be restored. It's needed for parsing the model I think.